db.js

```diff
@@ -95,14 +95,18 @@ var books = (function () {
 
       return Promise.resolve(retrievedBooks);
     },
-    get: async function (bookId) {
+    get: async function (bookId, injectLoanData) {
       var sql = "SELECT * FROM books WHERE id = ?";
       var inserts = [bookId];
       sql = mysql.format(sql, inserts);
 
       return pool.query(sql).then(([rows, fields]) => {
         if (!rows || rows.length === 0) {
           return null;
+        } else if (injectLoanData === true) {
+          return injectLoanInfo(rows[0]);
+        } else if (injectLoanData === false) {
+          return rows[0];
```

Comment:
On line 98, you should give the injectLoanData a default value of true.
`get: async function (bookId, injectLoanData = true) {`

This if statement can be simplified into `if (injectLoanData) {...} else {}`